src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -713,6 +716,9 @@
         - name: --auto-upgrade-channel
           type: string
           short-summary: Specify the upgrade channel for autoupgrade. It could be rapid, stable, patch, node-image or none, none means disable autoupgrade.
+        - name: --node-os-upgrade-channel
+          type: string
+          short-summary: <TODO let Paul fill in>
```

Comment:
Also this one

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/5a4f7662cba770d045d500894a5565c176453cb3/specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-11-02-preview/managedClusters.json#L5532

Same as https://github.com/Azure/azure-rest-api-specs/blob/5a4f7662cba770d045d500894a5565c176453cb3/specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-11-02-preview/managedClusters.json#L5532

"Manner in which the OS on your nodes is updated. It could be none, unmanged, nodeimage or securitypatch"

Reply:
Removed the default comment, as it might change